### PR TITLE
Temporary quota checking

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java [hunk 1]
```diff
@@ -61,6 +64,13 @@ public NodeResources decideNodeResources(Capacity capacity, ClusterSpec cluster)
         return resources;
     }
 
+    private boolean hasQuota(ApplicationId application, int requestedNodes) {
+        if ( ! this.zone.system().isPublic()) return true; // no quota management
+
+        if ("yj".equals(application.tenant().value())) return requestedNodes <= 60;
```
Comment (kkraune) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java]: Use a hash to anonymize?

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java [hunk 2]
```diff
@@ -26,10 +27,12 @@ public CapacityPolicies(Zone zone) {
...
-    public int decideSize(Capacity capacity, ClusterSpec.Type clusterType) {
-        int requestedNodes = ensureRedundancy(capacity.nodeCount(), clusterType, capacity.canFail());
+    public int decideSize(Capacity capacity, ClusterSpec.Type clusterType, ApplicationId application) {
+        int requestedNodes = capacity.nodeCount();
 
-        if (this.zone.system().isPublic() && requestedNodes > 5)
+        ensureRedundancy(requestedNodes, clusterType, capacity.canFail());
+
+        if ( ! hasQuota(application, requestedNodes))
             throw new IllegalArgumentException(requestedNodes + " exceeds your quota. Please contact Vespa support");
```
Comment (kkraune) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java]:
```suggestion
            throw new IllegalArgumentException(requestedNodes + " exceeds your quota, resolve using https://cloud.vespa.ai/quota");
```

**Review by kkraune — APPROVED** [review]

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java]: ok

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java (hunk 2 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java]: Good, thanks

**Pull request merged**